src/main/webapp/app/exercises/text/assess/text-assessment-dashboard/text-assessment-dashboard.component.ts

```diff
@@ -58,6 +60,13 @@ export class TextAssessmentDashboardComponent implements OnInit {
             });
     }
 
+    public sortRows() {
+        if (this.predicate === 'durationInMinutes') {
+            this.submissions.forEach((submission) => (submission.durationInMinutes = this.durationForSubmission(submission)));
```

Comment:
same comment as for file upload exercises

Reply:
Addressed via: 55d5b75